Support @ObjCName Swift Name in Enum

### First of all, congrats on make the tool open source :tada:. The Touchlab work for the KMP community is amazing! Great thanks for all the working of all you folks have being doing.

I was doing the first setup of Skie here on multiplatform project, it seems to have compilation issue in the generated Swift Extensions when trying to compile against a `enum value` that is using `@ObjcName(swiftName = "...")`. See the log below.

I also look into the [Known Issues page](https://skie.touchlab.co/KnownIssues) but did not find the comment about this issue, this is why I'm openning.
 
```
path/to/the/project/build/skie/releaseFramework/iosX64/swift/generated/com.sample.module.HomeNavigation.OnboardingStep.swift:32:165: error: 'tohome' has been renamed to 'toHome'
    case .tohome: return __Skie.class_path_to_file_HomeNavigation_OnboardingStep.tohome as __Skie.class___path_to_file_HomeNavigation_OnboardingStep
                                                                                                                                                ^~~~~~~
                                                                                                                                                 toHome
KotlinShared.__HomeNavigationOnboardingStep:7:20: note: 'tohome' was obsoleted in Swift 3
    open class var tohome: KotlinShared.HomeNavigationOnboardingStep { get }
```

The kotlin code:
```kotlin
public sealed interface HomeNavigation { 
    public enum class OnboardingStep : HomeNavigation {
        @ObjCName(swiftName = "toHome") ToHome;
    }
}
```

SKIE Setup:
```kotlin
skie {
    features {
        group {
            SealedInterop.Enabled(true)
            EnumInterop.Enabled(true)
            DefaultArgumentInterop.Enabled(true)
            FlowInterop.Enabled(false)
            SuspendInterop.Enabled(false)
        }
    }
}
```

--- a/SKIE/compiler/kotlin-plugin/src/kgp_common/kotlin/co/touchlab/skie/api/model/type/ActualKotlinEnumEntrySwiftModel.kt
+++ b/SKIE/compiler/kotlin-plugin/src/kgp_common/kotlin/co/touchlab/skie/api/model/type/ActualKotlinEnumEntrySwiftModel.kt
@@ -12,7 +12,7 @@ class ActualKotlinEnumEntrySwiftModel(
     private val swiftModelScope: MutableSwiftModelScope,
 ) : KotlinEnumEntrySwiftModel {
 
-    override val identifier: String = namer.getEnumEntrySelector(descriptor.original)
+    override val identifier: String = namer.getEnumEntrySwiftName(descriptor.original)
 
     override val enum: KotlinClassSwiftModel
         get() = with(swiftModelScope) {

